tutors3/patches/openedx-lms-common-settings

```diff
@@ -2,12 +2,9 @@
 PROFILE_IMAGE_BACKEND = {
     "class": DEFAULT_FILE_STORAGE,
     "options": {
-        "bucket": "{{ S3_PROFILE_IMAGE_BUCKET }}",
+        "bucket_name": "{{ S3_PROFILE_IMAGE_BUCKET }}",
         "location": PROFILE_IMAGE_BACKEND["options"]["location"].lstrip("/"),
         {% if S3_PROFILE_IMAGE_CUSTOM_DOMAIN %}"custom_domain": "{{ S3_PROFILE_IMAGE_CUSTOM_DOMAIN }}",{% endif %}
-        # A non empty base_url is necessary in development as it will be used
-        # to build a static url for the profile images.
-        "base_url": "dummyprofileimagebaseurl",
```

Comment:
I am not sure I understand why it's necessary to remove this from here, and then reindroduce it in `openedx-lms-development-settings`. Can you elaborate please?

Reply:
Sure. The comment indicates that this is necessary for development and I thought it was cleaner to move this to the dev-specific patch. 
This didn't cause an issue for me but I moved it for peace of mind.

Reply:
Okay, that makes sense. :)